frontend/www/js/omegaup/group/edit.ts

```diff
@@ -9,11 +9,24 @@ import T from '../lang';
 import Vue from 'vue';
 import * as CSV from '@/third_party/js/csv.js/csv.js';
 import {
-  cleanRecords,
+  downloadCsvFile,
   generateHumanReadablePassword,
   generatePassword,
+  getCSVRecords,
 } from '../groups';
 
+export type GroupCSVDatasetRecord = types.Identity;
+
+export const requiredFieldsGroups = new Set(['username']);
+
+export const optionalFieldsGroups = new Set([
```

Comment:
```suggestion
export const groupRequiredFields = new Set(['username']);

export const groupOptionalFields = new Set([
```

aquí es mejor usar "group" como prefijo para que queden más cerca conceptualmente.

Reply:
Listo